build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group = "com.bmuschko"
-version = "0.2"
+version = "0.3-SNAPSHOT"
```

Comment:
Let's revert this change. Version 0.2 hasn't been released yet so the version has to be 0.2.

Reply:
That’s odd, version 0.2 is visible from jcenter for quite a while

Reply:
Looks like I didn't update the documentation. My bet. So yeah, the next version is going to be 0.3. Can you either change this to 0.3? I am not going to publish a SNAPSHOT.